### ISSUE TITLE
houdini beta 3

### ISSUE THREAD

**Issue by neoo2005** [iabem97/houdini-public#45]
I am a developer account. But beta 3 is not available for download.
 Can you send me a ipa file by email?
houdini beta 3 to mail neoo2005@yahoo.com thank you

**Comment by karem49** [comment]
Do you have ipa file for houdini ?

**Comment by karem49** [comment]
B3!!

**Comment by karem49** [comment]
If someone have the ipa houdini b3 please send me one 
karem49@gmail.com

**Comment by theexoeb9632** [comment]
hey send me houdini beta 3 ipa
thee9632@gmail.com

**Comment by fizdog** [comment]
Stop lying about having a developer account lol

**Comment by karem49** [comment]
Stop lying???

**Comment by fizdog** [comment]
Yeah cause people say that then still want an ipa? Really. As much trouble I am having compiling I’m not asking for an ipa

**Comment by velcis** [comment]
Just clone and open with XCode, then change your account in app defaults. 
Easy, i`ve done but without a developer account.

**Comment by karem49** [comment]
Thanks

**Comment by Yashkigura** [comment]
Sir i need jailbreaker for ios 11.4 , send to me the ipa, Thanks 💯 yashhkigura28@gmail.com

**Comment by Antona12** [comment]
am a developer account. But beta 3 is not available for download.
There seems to be a problem. Can you send me a ipa file by email?
toha93l@mail.ru

**Comment by iM4CH3T3** [comment]
U can’t install through ipa. U need Xcode and your devices udid synced to your dev account.

**Comment by Yashkigura** [comment]
how?

**Comment by JunyuT** [comment]
Please stop asking for ipa.

If you really want it, compile it yourself. If you don't know how to, you most likely won't want this anyways. Houdini is not a full jailbreak, and you NEED the $99 dev account for it to work. (_Honestly if you really do have that account, you should know how to compile this. Otherwise, why tf do you have a $99 dev account when you can't even compile an xcode project??? Stop lying folks_)

You can get an ipa through ./compile.sh , otherwise you'd have to connect your phone to your computer and run the project through xcode.

**Comment by fizdog** [comment]
Some people have it to bypass the 7 day when sideloading apps 👍🏻. Also when I did ./compile.sh cydia impactor said it’s missing Info.plist

**Comment by mmtt88** [comment]
hey send me houdini beta 3 ipa
528073345@qq.com

**Comment by jaker1234** [comment]
985864327@qq.com
send me houdini beta 3 ipa plz thinks

**Comment by anhvanle** [comment]
Hey send me houdini beta3 ipa 
Ngoilangcuagio321@gmail.com

**Comment by fizdog** [comment]
Stop asking for the ipa you’re not going to get it. You need a developer account.

**Comment by 0snoopy0** [comment]
lol xD

**Comment by JunyuT** [comment]
go away no ipa here

**Comment by fizdog** [comment]
If you want ipa, get a dev account and compile it yourself 👍🏻

**Comment by souna9009** [comment]
hey please send me houdini beta 3 ipa i’m a developer : souna875449@gmail.com

**Comment by fizdog** [comment]
If you’re a dev like you say you are. Get on a Mac running high Sierra and compile it through Xcode. 👍🏻
